### PR TITLE
Updated UserCredentials Dialog to deal with QuickControls2

### PR DESCRIPTION
**Pull request by anmacdonald**
With the turning on of High-DPI mode, the user credentials dialog showed heavily inflated text.

This change has removed the scale-factor calculations, and upgraded the dialog to QuickControls 2. 

References to explicit widths/heights have been removed in favour of resizing to contents.

### CODE REVIEW

File: Import/Esri/ArcGISRuntime/Toolkit/Dialogs/UserCredentialsView.qml [hunk 1]
```diff
@@ -227,3 +222,5 @@ Rectangle {
     }
 }
 
+
+
```
Comment (JamesMBallard) [inline Import/Esri/ArcGISRuntime/Toolkit/Dialogs/UserCredentialsView.qml]: Please remove the empty lines.

**Comment by anmacdonald** [comment]
Looks good on Android and Mac. Happy to test on iOS/Windows!

**Comment by JamesMBallard** [comment]
>Looks good on Android and Mac. Happy to test on iOS/Windows!

@anmacdonald yes, please if you have time.

**Comment by anmacdonald** [comment]
Hi @JamesMBallard, I think these are acceptable! 

## Android

android_usercredentials

## iOS

ios_usercredentials

## MacOS

macos_usercredentials

## Windows

windows_usercredentials

**Comment by lsmallwood** [comment]
I think it would be worth putting a newline in before the URL so that (at least in the general arcgis.com case it is not split over 2 lines). So something like:

```
You need to sign in to access the
resource at:
'https://www.arcgis.com'
```

Also, any reason that we ha v e the URL in single quotes?

@ldanzinger, @JamesMBallard - any thoughts?

**Comment by JamesMBallard** [comment]
Testing is good. Code changes look good but it would be good to see if we can prevent the url from wrapping across lines as @lsmallwood mentioned. If that gets too hairy we can put that into a new issue to work on.

**Comment by anmacdonald** [comment]
If there is a consensus then shall add. Will add as a new ``Text { }`` to the layout rows. I don't trust newline characters in QML rendering based on how I've seen the callouts behave.

**Comment by anmacdonald** [comment]
New look authentication view:

screen shot 2018-11-08 at 14 08 46

**Pull request merged**